uint8 id
string location_source
string location_target
bool trigger_update
